aspnetcore/security/authentication/identity-enable-qrcodes.md

```diff
@@ -61,10 +73,16 @@ private string GenerateQrCodeUri(string email, string unformattedKey)
 }
 ```
 
-The second parameter in the call to `string.Format()` is your site name, taken from your solution name. It can be changed to anything value, but it must always be URL encoded.
+The second parameter in the call to `string.Format` is your site name, taken from your solution name. It can be changed to any value, but it must always be URL encoded.
 
 ## Using a different QR Code library
 
-You may have a preferred QR Code library which you want to use. The `qrCode` element in the view is a suitable place to put the QR code.
+You can replace the QR Code library in the `qrCode` element.  <!-- Original
```

Comment:
From the files changes tab, edit and merge directly into this commit. Or fix after merge
![image](https://user-images.githubusercontent.com/3605364/28837787-e2cc4216-76a2-11e7-8d3b-648e27d8c6b7.png)